roles/epel_repositories/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 epel_repositories_state: present
+epel_repositories_url: http://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
+epel_repositories_name: "{{ epel_repositories_url if epel_repositories_state == 'present' else 'epel-release' }}"
```

Comment:
CentOS extras also carries `epel-release` but this makes it work on RHEL as well so :+1: